db/migrate/20170213132040_remove_slug_migrations.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveSlugMigrations < ActiveRecord::Migration[5.0]
+  def up
+    drop_table :slug_migrations
+  end
+
+  def down
+    fail ActiveRecord::IrreversibleMigration
```

Comment:
I was debating making it reversible, but once the data's gone it seemed like a bit of a moot point…?